policies/linux.consul.-.catalog.nodes_critical.json

```diff
@@ -0,0 +1,28 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 3,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 1,
+        "metric": "consul.catalog.nodes_critical",
+        "operator": "<"
+      }
+    ],
+    "deleted": false,
+    "description": "consul.catalog.nodes_critical is greater than 0.",
```

Comment:
Typically, we explain why this alert should be paid attention. Can you provide more details in the description?